badge-maker/lib/color.js

```diff
@@ -49,7 +49,7 @@ function normalizeColor(color) {
   } else if (color in aliases) {
     return aliases[color]
   } else if (isHexColor(color)) {
-    return `#${color.toLowerCase()}`
+    return `#${color.toString().toLowerCase()}`
```

Comment:
How about moving the `toString` to `isHexColor()`? Like this below:

```
function isHexColor(s = '') {
  return hexColorRegex.test(String(s))
}
```

The `String(s)` is safer here. Even if the value is `null,` it will not throw an error.

Reply:
I don't think that would really help tbh.

The issue here is that the current implementation of `isHexColor()` may already return true on a non-string input (I guess `.test()` implicitly casts to string), leading us to try and call `toLowerCase()` on an int. For example:

```
> isHexColor(111111)
true
```

so even if we make the cast explicit, we'd still need to call `toString()` in this block, or do it somewhere before we reach here.

Null or undefined already return false with the current implementation:

```
> isHexColor(null)
false
> isHexColor(undefined)
false
```